### ISSUE TITLE
the same animals are not groupped by maps

### ISSUE THREAD

**Issue by mikuyourworld** [LordHansCapon/cotw-harvest-tracker#4]
like MULE DEER,
it lives in 3 maps, PARQUE FERNANDO, SILVER RIDGE PEAKS, RANCHO DEL ARROYO
you hunts one in SILVER RIDGE PEAKS,
but when you check MULE DEER in PARQUE FERNANDO,
the one you hunt in SILVER RIDGE PEAKS will be shown.

could you fix this? Thank you.

**Comment by LordHansCapon** [comment]
Yeah, currently it does not track per map but only per animal. This was a conscious decision, I currently have no intention to change it, but if enough request is made for this, maybe I'll do it. However, since no map is tracked, older harvests would be probably categorized to the first map they occur or something, as the tracker is not tracking the map.

**Comment by mikuyourworld** [comment]
And I also want to mention why I am particularly concerned about this feature. It's because that currently the tool only supports English. I am trying to localize this useful tool to support other languages. After trying to add support for Chinese, I encountered a problem. Sometimes, in non-English languages, different animal English names are translated into the same word. Like **WILD BOAR** and **FERAL PIG**, their name in Chines are the same, **野猪**. And the **PUMA** and **MOUNTAIN LION** are **美洲狮**. Therefore, if we don't classify the animals according to the map, issues may arise.

**Comment by LordHansCapon** [comment]
> And I also want to mention why I am particularly concerned about this feature. It's because that currently the tool only supports English. I am trying to localize this useful tool to support other languages. After trying to add support for Chinese, I encountered a problem. Sometimes, in non-English languages, different animal English names are translated into the same name. Like **WILD BOAR** and **FERAL PIG**, their name in Chines are the same, **野猪**. And the **PUMA** and **MOUNTAIN LION** are **美洲狮**. Therefore, if we don't classify the animals according to the map, issues may arise.

Ouch, yes, that's a shame. As far as I seen, there is no animal ID or any sort of stuff that is always the same for the same animal type, only the name, hence I used the name. I was aware of translational problems this may generate, but not on this scale.

**Comment by LordHansCapon** [comment]
Version 2.0 is on experimental branch and that version now tracks harvests per reserve as per requested.
